pkg/config/config.go

```diff
@@ -281,9 +283,6 @@ type EngineConfig struct {
 	// MachineEnabled indicates if Podman is running in a podman-machine VM
 	MachineEnabled bool `toml:"machine_enabled,omitempty"`
 
-	// MachineImage is the image used when creating a podman-machine VM
-	MachineImage string `toml:"machine_image,omitempty"`
```

Comment:
This field is not in podman 3.3, right? In this case I would want this PR and the podman wiring to get into podman 3.4 so that we do not have to worry about backwards compat for this field.

Reply:
Yes this is not in 3.3